src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -629,13 +629,6 @@ void *xocl_drm_init(xdev_handle_t xdev_hdl)
                 goto failed;
 	}
 
-        drm_p->cma_bank_idx = -1;
-	/* Memory manager initialization is done in two phase. 
-	 * Phase 1 is done based on platform and Phase 2 is done 
-	 * based on XCLBIN. We have done phase 1 here.
-	 */
-        drm_p->xocl_drm_mm_done = true;
-
```

Comment:
Can you explain why this code can be removed?

Reply:
I have sent a mail with the details.

Reply:
okay, looks like drm_p->xocl_drm_mm_done is not used anymore. Can we delete it in xocl_drm.h?